docs/upgrade/_common/upgrade-guide-v5-ubuntu-and-debian-p1.rst

```diff
@@ -2,49 +2,43 @@
 Upgrade Guide - |SCYLLA_NAME| |SRC_VERSION| to |NEW_VERSION| for |OS|
 =============================================================================
 
-This document is a step by step procedure for upgrading from |SCYLLA_NAME| |SRC_VERSION| to |SCYLLA_NAME| |NEW_VERSION|, and rollback to |SRC_VERSION| if required.
+This document is a step by step procedure for upgrading from |SCYLLA_NAME| |SRC_VERSION| to |SCYLLA_NAME| |NEW_VERSION|, and rollback to version |SRC_VERSION| if required.
 
-..
-  Relevant and tested for Ubuntu 20.04. Remove from other OSes and versions.
-
-There are two upgrade alternatives: you can upgrade ScyllaDB simultaneously updating 3rd party and OS packages (recommended for Ubuntu 20.04), or upgrade ScyllaDB without updating any external packages.
```

Comment:
I don't know what this paragraph referred to and whether it's still relevant. @tzach who in release engineering can also please review this document?

Reply:
@yaronkaikov !

Reply:
@nyh 
@yaronkaikov provided the information, and I followed the guidelines from this PR: https://github.com/scylladb/scylladb/pull/11460

I've removed that section because it was a bug. The information from the [upgrade-guide-v5-ubuntu-and-debian-p1.rst](https://github.com/scylladb/scylladb/pull/11577/files/2513497f9aa9d400a482a00745afcef210be5589#diff-657c5fc0bc3db9dd28d20f160d661ccc7a176b8a0b10ff1e0bc7694bf611f6a6) file is included in three upgrade guides: for Ubuntu, Debian, and ScyllaDB image, while the section about "upgrade alternatives" is only relevant for the image.
I moved it from that file to another file that is only included in the image upgrade guide.